.circleci/config.yml

```
@@ -37,10 +37,10 @@ jobs:
       - run:
           name: Insert key and flag data into demo
           command: |
-            sed -i "s/YOUR_SDK_KEY/${LD_HELLO_WORLD_SDK_KEY}/" hello.r
-            sed -i "s/YOUR_FEATURE_KEY/${LD_HELLO_WORLD_FLAG_KEY_WITH_TRUE_VALUE}/" hello.r
+            sed -i "s/SDK_KEY=*\".*\"/SDK_KEY=\"${LD_HELLO_WORLD_SDK_KEY}\"/" hello.r
```

An alternative syntax for sed might read slightly cleaner. But overall I think it's fine.

```suggestion
            sed -i "/^SDK_KEY=/c SDK_KEY=\"${LD_HELLO_WORLD_SDK_KEY}\"/" hello.r
```

---

I tried these two suggestions and kept getting build errors ([example](https://app.circleci.com/pipelines/github/launchdarkly/hello-r/59/workflows/810a8b4f-5810-46ec-9322-5b8f98e5a1ce/jobs/61), [example](https://app.circleci.com/pipelines/github/launchdarkly/hello-r/60/workflows/f91370bb-cc4a-4ef2-ac42-97bd6f9264f5/jobs/62)), which looked like they were perhaps because of an unclosed string left in the variable declaration (?? guessing here). I reverted back to the original syntax, which is now passing again.

I'm going to merge since you've approved, but let me know if you'd like additional followup here. Thanks!